libraries/disp/viewers/rtfiffrawview.cpp

```diff
@@ -347,7 +347,7 @@ void RtFiffRawView::setZoom(double zoomFac)
 {
     m_fZoomFactor = zoomFac;
 
-    m_pTableView->verticalHeader()->setDefaultSectionSize(m_fZoomFactor*m_fDefaultSectionSize);//Row Height
+    m_pTableView->verticalHeader()->setDefaultSectionSize(m_pTableView->height() / m_fZoomFactor/**m_fDefaultSectionSize*/);//Row Height
```

Comment:
should we keep the commented out `m_fDefaultSectionSize`?

Reply:
removed it, also removed m_fDefaultSectionSize entirely